libblkid/src/superblocks/zfs.c

```diff
@@ -58,7 +58,7 @@ struct nvuint64 {
 	uint32_t	nvu_type;
 	uint32_t	nvu_elem;
 	uint64_t	nvu_value;
-};
+} __attribute__((packed));
 
```

Comment:
The question is if all the structs should not be marked as "packed". I guess ZFS code writes it (the list) to the device as packed. We will see...